docs/pages/deployment/docker.rst

```diff
@@ -0,0 +1,129 @@
+.. _running-docker:
+
+Running on Docker
+#################
+
+If you already use Docker, the easiest way to get your Nuts Node up and running for development or production is
+using Docker. This guide helps you to configure the Nuts node in Docker.
+To use the most recent release use ``nutsfoundation/nuts-node:latest``. For production environments it's advised to use a specific version.
+
```

Comment:
can we start with the most important part when people start with Nuts, which is i.m.o. the Docker run and docker compose examples?

Then probably the dev image, and user/mount stuff

Reply:
reordered some stuff, does this work?